pkg/db/db.go

```diff
@@ -219,6 +223,90 @@ WHERE timestamp > NOW() - INTERVAL '14 DAY'
 GROUP BY tests.name, tests.id, date, release, job_name
 `
 
+type PostgresSetReturningFunction struct {
+	// Name is the name of the set returning function.
+	Name string
+	// Definition is the material view definition.
+	Definition string
+}
+
+func createPostgresFunctions(db *gorm.DB) error {
+	for _, pmv := range PostgresMatViews {
+		vd := pmv.Definition
+		for k, v := range pmv.ReplaceStrings {
+			vd = strings.ReplaceAll(vd, k, v)
+		}
+
+		if res := db.Exec(jobResultFunction); res.Error != nil {
+			klog.Errorf("error creating materialized view %s: %v", pmv.Name, res.Error)
+			return res.Error
+		}
```

Comment:
Is this meant to be in a "range PostgresMatViews"? It looks like we want to drop the for loop and just create this function. (possibly an array of functions someday if more are added)

Reply:
Oops.  I had intended to create an array of `PostgresSetReturningFunction`'s, but so far there's only one. The fix you pushed is good.